common/predictive-text/worker/defaultWordBreaker.ts

```diff
@@ -1,9 +0,0 @@
-class DefaultWordBreaker implements WorkerInternalWordBreaker {
-  // TODO:  Specify the data needed to implement a 'default' word breaker.
-  constructor(obj) {
```

Comment:
Do you have a corresponding PR ready for https://github.com/keymanapp/lexical-models?  If not, this will break the existing infrastructure.

A matching PR there would also make a good initial example of using the new structures.